dashboard/test/ui/features/teacher_tools/rubrics/teacher_view_of_rubric.feature

```diff
@@ -5,7 +5,7 @@ Scenario: Teachers can give and send feedback on the rubric to students.
   Given I create an authorized teacher-associated student named "Lillian"
 
   # Student can see the rubric and submit work
-  And I am on "http://studio.code.org/s/allthethings/lessons/48/levels/2?enableExperiments=ai-rubrics"
+  And I am on "http://studio.code.org/s/allthethings/lessons/48/levels/2"
```

Comment:
I haven't seen the drone results yet, but I would expect that you would need to use a verified teacher in order for the teacher to be able to view the new rubrics UI below

Reply:
I just checked drone, and there are also a lot of unexpected unit test failures from the frontend change pulling the teacher's verified status from redux. I'll hit these UI tests and the unit tests first thing tomorrow morning.